lib/views/widgets/Events/event_card.dart

```diff
@@ -0,0 +1,224 @@
+                                fontWeight: FontWeight.bold,
+                                fontSize: totalHeight * 0.0141,
+                                fontFamily:
+                                    GoogleFonts.montserrat().fontFamily),
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                )
+              : Container(
+                  child: Row(
+                    children: [
+                      Expanded(
+                          child: SizedBox(
+                            height: 107,
+                            width: 107,
+                            child: Image(
+                              image: AssetImage(
+                                  'assets/images/saaranga logo 1.png'),
+                            ),
+                          ),
+                          flex: 1),
+                      SizedBox(width: 25),
+                      Expanded(
+                        child: Column(children: [
+                          Expanded(
+                              child: Align(
+                                alignment: Alignment.centerLeft,
+                                child: Text(
+                                  widget.event.name,
+                                  style: TextStyle(
+                                      fontWeight: FontWeight.bold,
+                                      fontSize: totalHeight * 0.03,
+                                      fontFamily:
+                                          GoogleFonts.montserrat().fontFamily),
+                                ),
+                              ),
+                              flex: 1),
+                          Expanded(
+                              child: Align(
+                                alignment: Alignment.centerLeft,
+                                child: Text(
+                                    widget.event.r1Date +
+                                        '\n' +
+                                        widget.event.r2Date,
+                                    style: TextStyle(
+                                        fontWeight: FontWeight.bold,
+                                        fontSize: totalHeight * 0.015,
+                                        fontFamily: GoogleFonts.montserrat()
+                                            .fontFamily)),
+                              ),
+                              flex: 1),
+                          Expanded(
+                              child: Align(
+                                alignment: Alignment.centerLeft,
+                                child: Text(
+                                  'Know more',
+                                  style: TextStyle(
+                                      fontFamily:
+                                          GoogleFonts.montserrat().fontFamily,
+                                      fontSize: totalHeight * 0.012),
```

Comment:
The font size here really makes the know more hidden.maintain a constant visible font-size